entrypoint.sh

```diff
@@ -7,14 +7,15 @@ APP_PATH="/home/celestia/.celestia-app"
 NODE_PATH="/home/celestia/bridge/"
 
 # Check if the folder exists
+echo "🔍 Checking if the folder $APP_PATH exists..."
```

Comment:
This is an excellent way to ensure that most users whose terminals do not support UTF-8 (and that includes `urxvt`, `gnome-terminal`, `mate-terminal` most editors etc. will see something which potentially makes them ask questions. I'd recommend not including these things if possible.

Reply:
good call, thank you @appetrosyan. done in https://github.com/rollkit/local-celestia-devnet/pull/63/commits/325e3f7691ee59dc0c78415b1d4365dda1107ea3